src/traffic_manager/traffic_manager.cc

```diff
@@ -179,7 +179,7 @@ is_server_idle_from_new_connection()
 
   Debug("lm", "%" PRId64 " active clients, threshold is %" PRId64, active, threshold);
 
-  return active <= threshold;
+  return true;
 }
```

Comment:
I suggest:
```
static bool
is_server_idle_from_new_connection()
{
  RecInt threshold = 0;

  Debug("lm", "%" PRId64 " active clients, threshold is %" PRId64, active, threshold);

#if 0
  // TODO implement with the right metric
  RecInt active    = 0;
  return active <= threshold;
#else
  return true;
#endif
}
```
This is clearer, and will avoid any potential unused local variable warnings.